docs/network-policy.md

```diff
@@ -0,0 +1,149 @@
+# Cluster Network Policy
+
+ClusterNetworkPolicy is a specification of how workloads within a cluster
+communicate with each other and other external endpoints.
+The ClusterNetworkPolicy is supposed to aid cluster admins to configure
+the security policy for the cluster, unlike K8s NetworkPolicy, which is
+aimed towards developers to secure their apps and affects Pods within the
+Namespace in which the K8s NetworkPolicy is created.
+Rules belonging to ClusterNetworkPolicies are evaluated before any rule
+belonging to K8s NetworkPolicy.
```

Comment:
s/to K8s NetworkPolicy/ to a K8s NetworkPolicy

Reply:
done